federation/README.md

```diff
@@ -0,0 +1,318 @@
+
+```
+broker> spire-server experimental bundle set -id spiffe://stocksmarket.org -path /some/path/stocksmarket.org.bundle
+```
+
+Now both SPIRE Servers can validate SVIDs having SPIFFE IDs with each other's trust domain, thus both can start fetching trust bundle updates from each other's federation endpoints. Also, as of now they can create registration entries for federating.
+
+Something important to consider is that the creation of the `broker.org.bundle` file (and later setting by the stocks market service) is not needed when the broker's SPIRE Server is using WebPKI authentication for its federation bundle endpoint.
+
+# Create registration entries for federation
+
+Now that they have each other trust bundle, let's see how they can create registration entries to federate with each other.
+
+To simplify things, we are going to suppose that the webapp and the quotes service are running both on Linux boxes, one owned by the broker and the other owned by the stocks market organization. Since they are using SPIRE, each Linux box also has a SPIRE Agent installed. In addition to this, the webapp is run using the `webapp` user, and the quotes service is run using the `quotes-service` user.
+
+Then, in the SPIRE Server node of the broker, they must create a registration entry as follows:
+
+```
+broker> spire-server entry create \
+	-parentID <SPIRE Agent's SPIFFE ID> \
+	-spiffeID spiffe://broker.org/webapp \
+	-selector unix:user:webapp \
+	-federatesWith "spiffe://stocksmarket.org"
+```
+
+Once this registration entry is created, when the webapp asks for an SVID it will get one having the `spiffe://broker.org/webapp` identity, along with the trust bundle associated to the `stocksmarket.org` trust domain, due to the use of the `-federateWith` flag when creating the registration entry. Worth to mention that if the `-federateWith` flag would not be used, federation would not work.
+
+At the stocks market service side, they must create a registration entry as follows:
+
+```
+stocks-market> spire-server entry create \
+	-parentID <SPIRE Agent's SPIFFE ID> \
+	-spiffeID spiffe://stocksmarket.org/quotes-service \
+	-selector unix:user:quotes-service \
+	-federatesWith "spiffe://broker.org"
+```
+
+Similarly, once this registration entry is created, when the quotes service asks for an SVID it will get one having the `spiffe://stocksmarket.org/quotes-service` identity, along with the trust bundle associated to the `broker.org` trust domain.
+
+That is about it, now all the pieces are together to make federation work and see how the webapp is able to communicate with the quotes service despite having identities with different trust domains.
+
+# Federation Example using SPIFFE authentication with SPIRE 0.11.0
+
+This section explains how to try the example implementation of the scenario described previously.
+
+## Requirements
+
+- Go 1.14
+- docker-compose
```

Comment:
I copied this from Andrés' nested SPIRE tutorial. Is the new, first bullet correct?
```suggestion
## Requirements

Required files for this tutorial can be found in the `nested-spire` directory in https://github.com/spiffe/spire-tutorials. If you didn't already clone the repository please do so now.

Before proceeding, review the following system requirements:
- A 64-bit Linux or macOS environment
- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed (Docker Compose is included in macOS Docker Desktop)
- [Go](https://golang.org/dl/) 1.14.4 or higher installed
```

Reply:
Yes, it is correct, I will add it. Thanks!